### PR TITLE
Migrate from Carbon

### PR DESCRIPTION
**Pull request by splatteredbits**
There will be a future PR with new functionality and other changes. This is just to get the migration to a separate repo done. When merged, this version will not be published/deployed.

### CODE REVIEW

File: Carbon.IIS/Carbon.IIS.psm1 [hunk 1]
```diff
@@ -14,12 +14,54 @@
...
+        $microsoftWebAdministrationPath = Join-Path -Path $moduleRoot -ChildPath 'bin\Microsoft.Web.Administration.dll'
+    }
+    
+    if( (Test-Path -Path $microsoftWebAdministrationPath) )
+    {
+        Add-Type -Path $microsoftWebAdministrationPath
+    }
+}
+
+if( -not (Test-CTypeDataMember -TypeName 'Microsoft.Web.Administration.Site' -MemberName 'PhysicalPath') )
```
Comment (decoyjoe) [inline Carbon.IIS/Carbon.IIS.psm1]: Where does `Test-CTypeDataMember` come from? I don't see any nested modules being imported.

**Review by decoyjoe — APPROVED** [review]

File: Carbon.IIS/Carbon.IIS.psm1 (hunk 1 above)
Comment (splatteredbits) [reply Carbon.IIS/Carbon.IIS.psm1]: Whoops. That's from Carbon. I'll take care of it.

Comment (splatteredbits) [reply Carbon.IIS/Carbon.IIS.psm1]: PR for Carbon.Core so it contains this and other type data helper functions: https://github.com/webmd-health-services/Carbon.Core/pull/3

Comment (decoyjoe) [reply Carbon.IIS/Carbon.IIS.psm1]: Reviewed.

Comment (splatteredbits) [reply Carbon.IIS/Carbon.IIS.psm1]: Carbon.Core is now included as a private module.

**Pull request merged**